public/app/core/services/backend_srv.ts

```diff
@@ -524,15 +521,9 @@ export class BackendSrv implements BackendService {
     return this.get<DashboardDTO>(`/api/public/dashboards/${uid}`);
   }
 
+  /** @deprecated */
```

Comment:
we could remove these deprecations, but enough tests have mocked responses i think we can do that in an additional pass